readthedocs/doc_builder/exceptions.py

```diff
@@ -59,6 +59,8 @@ class BuildMaxConcurrencyError(BuildEnvironmentError):
 
 class DuplicatedBuildError(BuildEnvironmentError):
     message = ugettext_noop('Duplicated build.')
+    exit_code = 1
+    status_code = 0
```

Comment:
This should be a constant probably, and something other than 0?

Reply:
I made an `Enum` for this. I don't think it's a problem if it's 0. Why did you think it's better to have it different than 0? and what value?

Reply:
Because 0 to me means "normal" status -- we want a status that represents the issue of "duplicated build", which should be assigned a specific number.